### ISSUE TITLE
Substrate metadata -> Storage, Events are missing parameters and descriptions

### ISSUE THREAD

**Issue by SaltyCucumber** [polkadot-js/docs#345]
How it is:
Storage
![Screenshot 2022-09-07 at 19 09 32](https://user-images.githubusercontent.com/748932/188927320-99890672-31f1-4f90-9d7d-5f750d591b57.png)
Events
![Screenshot 2022-09-07 at 18 29 40](https://user-images.githubusercontent.com/748932/188927110-390035ea-4c02-41a7-9829-e1a138be364c.png)

How it should be:
![Screenshot 2022-09-07 at 18 43 03](https://user-images.githubusercontent.com/748932/188927098-35d5c8b0-6bc2-46f9-8dd9-5913becdcfb6.png)

**Comment by SaltyCucumber** [comment]
Got it, thanks. Will move to the source of the problem with this.

**Comment by jacogr** [comment]
Same with params for e.g. storage - those are (mostly) actually not documented in the Rust code, which means it doesn't make it to metadata, which means as part of the extract here it is not visible. 

And we certainly don't want to get back to Polkadot poc-1 level where we needed to duplicate EVERYTHING on the JS side, this was pre metadata. So the metadata is the only source of truth - what is there makes it in, what is not there, doesn't. (I needed to track each and every PR on Substrate and duplicate the values into the JS codebase)
